docs/challenge-003.md

```diff
@@ -2,8 +2,29 @@
 
 ## Challenge Objective
 
+For the last challenge, we want to write the aggregations from [Challenge 2](./challenge-002.md) into some "master data collection" (MDC). Again, we'll use a Cosmos DB collection as the MDC. After each store entity is finished aggregating the result, each item's aggregation should be written as either a new document in the MDC (for the first time we see that item) or as an update to an existing document.
+
+Ultimately, this MDC will be used to quickly retrieve aggregations via some sort of API (which is beyond the scope of this challenge). The objective is to make sure the MDC is kept up-to-date with accurate inventory data.
+
 ## Brief Guidance
 
+This challenge will focus on the red outlined area of the below design:
+
+![Architecture](https://user-images.githubusercontent.com/1093738/82704038-b9f77b80-9c42-11ea-8f64-a88fba66147c.jpeg)
+
+To accomplish this challenge, you will use the `mdc` container that you created in [Challenge 1](./challenge-001.md) to store inventory data that the orchestrator receives after calling an entity to perform some aggregation.
```

Comment:
I think we need to provide some logic somewhere in the challenges on how the change feed need to map to updates to the master inventory. Should it be in the solution design document? This document is our last opportunity to provide that guidance.

Reply:
We'll do it in the solution design document.